domain/dagconfig/consensus_defaults.go

```diff
@@ -76,4 +74,6 @@ const (
 	defaultTargetTimePerBlock = 1 * time.Second
 
 	defaultPruningProofM = 1000
+
+	defaultDeflationaryPhaseDaaScore = 15778800 // Half a year
```

Comment:
From my calculations this should be triggered at 10/6/2022, which is not 6 months from mainnet start (7/11/2021)

Reply:
Updated the calculation and added a comment